### PR TITLE
Create projects and about sections

### PR DESCRIPTION
**Pull request by whid-bb**
I have added two new sections:
- my project list
- about, that includes also my skills
Projects and about boxes were created using flexbox and grid

Bulb icon from about skills box was downloaded from:
[](url)https://www.flaticon.com/free-icons/idea

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,22 +26,236 @@
...
+                    <div>
+                        <h4>Skills</h4>
+                        <ul>
+                            <li>Codekit</li>
+                            <li>Codepen</li>
+                            <li>GitHub</li>
+                            <li>GitLAb</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
```
Comment (gemmen29) [inline index.html:250]: - [ ] Kindly wrap `Languages`, `Frameworks`, and `Skills` inside the `ul` tag and each one of them is a `li` tag for semantic purposes.

File: index.html [hunk 2]
```diff
@@ -25,22 +26,236 @@
...
+                                <li>JavaScript</li>
+                            </ul>
+                            <div class="project-preview-btn">
+                                <button type="button" class="btn-white-bg btn-white-dark-bg">
+                                    <span>See this project</span>
+                                    <img src="assets/img/arrow-right.png" alt="">
+                                </button>
+                            </div>
+                        </div>
+                    </div>
```
Comment (gemmen29) [inline index.html:76]: - [ ] Kindly make the projects take the full width in `375px` to match the Figma design.

| Your design | Figma design |
| --- | --- | 
| image |image |

File: index.html [hunk 3]
```diff
@@ -25,22 +26,236 @@
...
+                </button>
+                <img src="assets/img/image_geometry_2.png" alt="">
+            </div>
+        </section>
+        <section class="site-section ">
+            <div class="section-content">
+                <header>
+                    <h2>
+                        Projects
+                    </h2>
```
Comment (gemmen29) [inline index.html:53]: - [ ] Kindly add `font-weight: normal` for this heading to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 4]
```diff
@@ -25,22 +26,236 @@
...
+                    <!-- END project item -->
+                </div>
+            </div>
+        </section>
+        <section class="site-section">
+            <div class="section-content">
+                <header>
+                    <h2>
+                        About
+                    </h2>
```
Comment (gemmen29) [inline index.html:196]: - [ ] Kindly add `font-weight: normal` for this heading to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @whid-bb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (whid-bb) [reply index.html:76]: @gemmen29 Hello George! I copied the exact width from that template 320px. I can't find anything on that template mobile version that says 375px.

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved ✔️
Hi @whid-bb , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (gemmen29) [reply index.html]: @whid-bb The purpose of that exercise is to design the portfolio on a mobile view that starts from "375px". The portfolio is to be designed from the Figma template but you don't have to stick to the pixels of widths and heights from there.
As an example in desktop view in the Figma template are made on "1440px" but the desktop view milestone tells you to stick to "768px - 1100px".

Comment (gemmen29) [reply index.html]: @whid-bb You can use width with percentage it will solve the problem.

**Pull request merged**